README.md

```diff
@@ -1,5 +1,87 @@
+
+According to the data compilers, it was sourced from various sources, primarily government agencies. 
+
+They are : 
+National Center for Health Statistics - Mortality Files
+Behavioral Risk Factor Surveillance System
+National Center for Health Statistics - Natality files
+United States Diabetes Surveillance System
+USDA Food Environment Atlas, Map the Meal Gap from Feeding America
+Business Analyst, Delorme map data, ESRI, & US Census Tigerline Files
+Fatality Analysis Reporting System
+National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention
+Small Area Health Insurance Estimates
+Area Health Resource File/American Medical Association
+Area Health Resource File/National Provider Identification file
+CMS, National Provider Identification
+Mapping Medicare Disparities Tool
+State-specific sources & EDFacts
+American Community Survey, 5-year estimates
+Bureau of Labor Statistics
+Small Area Income and Poverty Estimates
+County Business Patterns
+Uniform Crime Reporting - FBI
+Environmental Public Health Tracking Network
+Safe Drinking Water Information System
+Comprehensive Housing Affordability Strategy (CHAS) data
+
+Full Feature List:
+Percent_Fair_or_Poor_Health
+Average_Number_of_Physically_Unhealthy_Days	
+Average_Number_of_Mentally_Unhealthy_Days	
+Percent_Smokers	Percent_Adults_with_Obesity	
+Food_Environment_Index	
+Percent_Physically_Inactive	
+Percent_With_Access_to_Exercise_Opportunities	
+Percent_Excessive_Drinking	
+Percent_Uninsured	
+Primary_Care_Physicians_Rate	
+Dentist_Rate	
+Mental_Health_Provider_Rate	
+Preventable_Hospitalization_Rate	
+Percent_Vaccinated	
+High_School_Graduation_Rate	
+Percent_Some_College	
+Percent_Unemployed	
+Percent_Single_Parent_Households	
+Social_Association_Rate	
+Violent_Crime_Rate	
+Polution_Average_Daily_PM2.5	
+Presence_of_Water_Violation	
+Percent_Severe_Housing_Problems	
+Percent_Drive_Alone_to_Work	
+Percent_Long_Commute	
+Percent_Adults_with_Diabetes	
+Percent_Limited_Access_to_Healthy_Foods	
+Median_Household_Income
+
+The raw data provided more features than these, but we deemed some irrelevant to the questions we wanted to ask.
+
+ Additionally, we augmented to these data by including county population. These data were sourced from the USDA Economic Reaearch Service.
+
+# Question to Investigate
+
+    1. Which factors are most significant in Heart Disease prevelance?
+
+        1a.Are environmental, lifestyle or health measures more impactful?
+
+    2.Which areas of the country suffer greatest with respect to heart disease?
+
+        2a. Which counties fair better?
+
+
+
+
+
```

Comment:
Should add a section for Design and add the Database section and talk about the Data Model we have with link to the ERD https://github.com/Khanguye/HeartDiseaseByCounty_ML/blob/master/model/diagram/HeartDisease_StateCounty.png